finat/point_set.py

```diff
@@ -53,13 +53,13 @@ def __init__(self, point):
         assert len(point.shape) == 1
         self.point = point
 
-    @property
+    @cached_property
     def points(self):
         # Make sure we conform to the expected (# of points, point dimension)
         # shape
         return self.point.reshape(1, -1)
 
-    @property
+    @cached_property
     def indices(self):
         return ()
```

Comment:
Any particular reason for these changes?

Reply:
I guess no reason not to, although note that you can replace this complicated indices thing by just `indices = ()`.

Reply:
These are `@abstractproperty` in `class AbstractPointSet(metaclass=ABCMeta)` so you have to write
```python
    @cached_property
    def indices(self):
        pass
```
if you set it at initialisation so it all seems much-of-a-muchness